blaze-core/src/main/scala/org/http4s/blazecore/util/Http1Writer.scala

```diff
@@ -0,0 +1,28 @@
+package org.http4s
+package blazecore
+package util
+
+import java.nio.ByteBuffer
+import java.nio.charset.StandardCharsets
+import scala.concurrent._
+
+import fs2._
+import org.http4s.blaze.http.Headers
+import org.http4s.blaze.pipeline.TailStage
+import org.http4s.blaze.http.http20.NodeMsg._
+import org.http4s.util.StringWriter
+import org.http4s.util.chunk._
+
+trait Http1Writer extends EntityBodyWriter {
```

Comment:
I think this concern spans more than just `HTTP/1.x` so perhaps another name is in order.

Reply:
It's named thus because Http2Writer needs real `Header`s, to create the frame, whereas we have some optimizations passing around the mutable StringWriter.  But you're not wrong.